cpp/examples/basic/README.md

```diff
@@ -0,0 +1,31 @@
+# Basic Standalone libcudf C++ application
+
+This simple C++ example demonstrates a basic libcudf use case and provides a
+minimal example of building your own application based on libcudf using CMake.
+
+The example source code loads a csv file that contains stock prices from 4
+companies spanning across 5 days, computes the average of the closing price
+for each company and writes the result in csv format.
+
+## How to compile and execute
+
+Prerequisites:
+- Cudatookit 11.0 or later
+- gcc-9
+- cmake 3.18
+- `libboost-filesystem-dev` and `zlib1g-dev`
```

Comment:
I'm not sure if it's this project's responsibility to account for prerequisites here or maybe I should let users refer to `Contributing.md`?

Reply:
Well, the prereqs are different if they are relying on a pre-built libcudf.